tables/Makefile.am

```diff
@@ -71,9 +71,13 @@ table_files = \
 	de-eurobrl6u.dis \
 	de-g0-core.uti \
 	de-g0.utb \
+	de-g0-bd-core.uti \
+	de-g0-bd.utb \
```

Comment:
"bidi" makes more sense as an abbreviation of "bidirectional" than "bd" so I think I will renamed the files. But you don't have to worry about that Bue.

Reply:
Done